arch/armv7a/signal.S

```diff
@@ -20,15 +20,13 @@
 .globl _signal_trampoline
 .type _signal_trampoline, %function
 _signal_trampoline:
-	// stack: return address, signal number
-	push {r0-r3,r9,r12,lr}
-	mrs r0, apsr
-	push {r0}
-	ldr r0, [sp, #32]
+	/* Signal number, cpu_context * */
+	pop {r0,r4}
```

Comment:
Hmm, I guess this has not been updated yet when alignment has been added in kernel

Reply:
It has, the reglist contains `,` not `-`. Same with `armv7m`